nixos/modules/programs/atop.nix

```diff
@@ -12,11 +12,83 @@ in
 
   options = {
 
-    programs.atop = {
+    programs.atop = rec {
 
+      enable = mkEnableOption "Atop";
+
+      package = mkOption {
+        type = types.package;
+        default = pkgs.atop;
+        description = ''
+          Which package to use for Atop.
+        '';
+      };
+
+      netatop = {
+        enable = mkOption {
+          type = types.bool;
+          default = false;
+          description = ''
+            Whether to install and enable the netatop kernel module.
+            Note: this sets the kernel taint flag "O" for loading out-of-tree modules.
+          '';
+        };
+        package = mkOption {
+          type = types.package;
+          default = config.boot.kernelPackages.netatop;
```

Comment:
`detaultText` attribute missing for a package.

This can lead to breakage in some evals:

```
error: attribute 'netatop' missing, at /nix/store/md569yk7k26jdknlbarfyai4i1lb7kkj-nixexprs.tar.xz/nixos/modules/programs/atop.nix:38:21
```

Reply:
Hey @samueldr, sorry I broke your website build :-(
I'm opening a PR momentarily adding defaultTexts.
I'm having a problem with the tests locally, but I think I've got a fix already.

Reply:
https://github.com/NixOS/nixpkgs/pull/123926

Reply:
No biggie about the build failure :). It's an unusual failure type that, for this particular package, won't be a problem on the Hydra for NixOS itself. So it's really not obvious that it can cause some fringe breakage.